packages/fluentui/react-northstar/src/utils/mergeProviderContexts.ts

```diff
@@ -1,34 +1,35 @@
-import { Renderer, StylesContextPerformance, StylesContextPerformanceInput } from '@fluentui/react-bindings';
-import { mergeThemes } from '@fluentui/styles';
+import { StylesContextPerformance, StylesContextPerformanceInput } from '@fluentui/react-bindings';
+import { mergeThemes, CreateRenderer, Renderer } from '@fluentui/styles';
 
 import { ProviderContextPrepared, ProviderContextInput } from '../types';
-import { createRenderer, felaRenderer } from './felaRenderer';
 import isBrowser from './isBrowser';
 
-const registeredRenderers = new WeakMap<Document, Renderer>();
+const defaultDocument = { document: 'document' };
+const registeredRenderers = new WeakMap<Document | typeof defaultDocument, Renderer>();
 
-export const mergeRenderers = (current: Renderer, next?: Renderer, target?: Document): Renderer => {
-  if (next) {
-    return next;
-  }
+export const getOrCreateRenderer = (createRenderer: CreateRenderer, target?: Document): Renderer => {
```

Comment:
Not sure about naming there, but don't have better ideas.

Reply:
Just `getRenderer` should be fine in my opinion. We don't have to expose in the name the fact that we are creating it if it doesn't exists..

Reply:
Renamed to `getRenderer`